releasePipeline/prerelease.md

```diff
@@ -11,21 +12,29 @@ It may be beneficial to complete a pre-release before starting a vx.xx.x release
 4. Ensure the Tekton CLI is installed. You can download it [here](https://tekton.dev/docs/cli/).
 5. Authenticate to the cicsk8s cluster using `cicsk8s sso`
 
+## Pre-release steps - Automated
+
+1. Ensure you have completed the [set up](#set-up) before continuing.
+2. Run [01-run-pre-release.sh](./01-run-pre-release.sh). If the process fails at any stage, you can continue by re-running the script that failed from the manual steps and finish using the [manual steps below](#pre-release-steps---manual).
+3. Run [25-check-artifacts-signed.sh](./25-check-artifacts-signed.sh). When prompted, choose the '`pre-release`' option.
+    - Each maven artifact should contain a file called com.auth0.jwt-<*VERSION*>.jar.asc. If the .asc files aren't present, debug and diagnose why the artifacts have not been signed.
+
+4. Send the [mvp image](https://development.galasa.dev/prerelease/maven-repo/mvp/dev/galasa/galasa-isolated-mvp) to Will Yates to perform the MEND scan to check for any vulnerabilities before moving onto the release process.
```

Comment:
Minor point but let's add my name in here as well as Will, as if he is not available to do this, I can.